schema/deploy/operations/schema.sql

```diff
@@ -0,0 +1,9 @@
+-- Deploy seattleflu/id3c-customizations:operations/schema to pg
+
+begin;
+
+create schema operations;
+
+comment on schema operations is 'Data supporting opertions management processes';
```

Comment:
Gasp! A typo in the comment ...supporting operations.

Reply:
🙃 Thanks!